test/scenarios-periodics/el92-src@offline.sh

```diff
@@ -1,18 +1,19 @@
 #!/bin/bash
 
 # Sourced from scenario.sh and uses functions defined there.
-
 scenario_create_vms() {
-    prepare_kickstart host1 kickstart-offline.ks.template ""
-    # Create a VM with embedded container images and 0 NICs
-    launch_vm host1 rhel-9.2-microshift-source-isolated "" "" "" "" 0
+    prepare_kickstart "host1" kickstart-offline.ks.template ""
+    #         vmname         boot_blueprint                      network_name vm_vcpus vm_memory vm_disksize vm_nics
+    launch_vm "host1"  rhel-9.2-microshift-source-isolated ""           ""       ""        ""          0
```

Comment:
Nit: we do not usually align and comment on arguments. We have a task to make those position independent

Reply:
+1, I'll remove them